docs/updating/migration-from-ckeditor-4.md

```diff
@@ -1,735 +1,110 @@
 ---
 # Scope:
-# Make the migration guide more discoverable 
+# Support migration from CKEditor 4 to CKEditor 5.
 
-category: updating
+category: ckeditor4-migration
 order: 10
-modified_at: 2022-11-09
+modified_at: 2023-03-17
 ---
 
 # Migration from CKEditor 4
 
 <info-box hint>
-This is a guide for users who would like to switch from CKEditor 4 to CKEditor 5. It assumes you already have some basic knowledge about CKEditor 5, as this is necessary to follow this guide. If that is not the case, please roam around the {@link installation/index Getting started} section to make youself familiar with the concepts and ideas behing this new, modern installment of our WYSIWYG editor. For example, feel free to explore our {@link features/index features page} to compare the available plugins with your needs.
+	<strong>CKEditor 4 reaches its End of Life (EOL) in June 2023</strong>. From then on, it will receive no more updates, new features, bug fixes, and most importantly, security patches.
 
-Before starting, decide what kind of installation to use for your new editor. Refer to the {@link installation/index Getting started with CKEditor 5} overview guide for more information on available installation methods.
+	This section of the documentation explains how to migrate to CKEditor 5.
+
+	If you have an active CKEditor 4 subscription and need help, you can contact [our support team](https://ckeditor.com/contact/). If you are not ready to migrate yet, for a limited time we are offering a paid Extended Support Model that will protect you against security vulnerabilities and/or breaking third-party API changes.
 </info-box>
 
-When compared to its predecessor, CKEditor 5 should be considered **a totally new editor**. Every single aspect of it was redesigned &mdash; from installation, to integration, to features, to its data model, and finally to its API. Therefore, moving applications using a previous CKEditor version to version 5 cannot be simply called an "upgrade". It is something bigger, so the "migration" term fits better.
+When compared to its predecessor, CKEditor 5 should be considered **a totally new editor**. Every single aspect of it was redesigned &mdash; from installation, to integration, to features, to its data model, and finally to its API.
 
-There is no "drop in" solution for migrating. In this guide we hope to summarize the most important aspects you need to consider before you proceed with installing CKEditor 5.
+There is no automatic solution for migrating. This section summarizes the most important aspects you need to consider before you proceed with installing CKEditor 5.
```

Comment:
```suggestion
There is no automatic solution for migrating. This section summarizes the most important aspects you need to consider before you proceed with moving to CKEditor 5.
```

Reply:
I don't think all of that can be done as a comparison table. CKE5 has some stuff whereas CKE4 has none, so nothing compares.